WcaOnRails/spec/models/competition_spec.rb

```diff
@@ -598,4 +598,15 @@
       expect(competition.competitors.count).to eq 2
     end
   end
+
+  context "searching" do
```

Comment:
It's nice of context message matches the method name: `#contains?` (`#` for instance method).